modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,203 @@
+
+    iou_avg = []
+    pr_avg = []
+    n_pr_avg = []
+
+    # Loop for every tracker
+    for tracker_name in trackers:
+
+        print("Tracker name: ", tracker_name)
+
+        number_of_thresholds = 21
+        iou_video = np.zeros(number_of_thresholds)
+        pr_video = np.zeros(number_of_thresholds)
+        n_pr_video = np.zeros(number_of_thresholds)
+        iou_thr = np.linspace(0, 1, number_of_thresholds)
+        pr_thr = np.linspace(0, 50, number_of_thresholds)
+        n_pr_thr = np.linspace(0, 0.5, number_of_thresholds)
+
+        # Loop for every video
+        for video_name in list_of_videos:
+
+            tracker, frames_for_reinit = init_tracker(tracker_name)
+            init_once = False
+
+            print("\tVideo name: " + str(video_name))
+
+            # Open specific video and read ground truth for it
+            gt_file = open(os.path.join(args.dataset, video_name,
+                                        "groundtruth.txt"), "r")
+            gt_bb = gt_file.readline().rstrip("\n").split(",")
+            init_bb = tuple([float(b) for b in gt_bb])
+
+            video_sequence = sorted(os.listdir(os.path.join(
+                args.dataset, video_name, "img")))
+
+            iou_values = []
+            pr_values = []
+            n_pr_values = []
+            frame_counter = len(video_sequence)
+
+            # Loop for every frame in the video
+            for number_of_the_frame, image in enumerate(video_sequence):
+                frame = cv.imread(os.path.join(
+                    args.dataset, video_name, "img", image))
+                gt_bb = tuple([float(x) for x in gt_bb])
+
+                # Check for presence of the object on the image
+                # Image is ignored if no object on it
+                if gt_bb[2] == 0 or gt_bb[3] == 0:
+                    gt_bb = gt_file.readline().rstrip("\n").split(",")
+                    continue
+
+                # Condition for reinitialization of the tracker
+                if ((number_of_the_frame + 1) % frames_for_reinit == 0):
+
+                    tracker, frames_for_reinit = init_tracker(tracker_name)
+                    init_once = False
+                    init_bb = gt_bb
+
+                if not init_once:
+                    init_state = tracker.init(frame, init_bb)
+                    init_once = True
+                init_state, new_bb = tracker.update(frame)
+
+                if args.visualization:
+                    cv.rectangle(frame, (int(new_bb[0]), int(new_bb[1])), (int(
+                        new_bb[0] + new_bb[2]), int(new_bb[1] + new_bb[3])), (
+                            200, 0, 0))
+                    cv.imshow("Tracking", frame)
+                    cv.waitKey(1)
+
+                iou_values.append(get_iou(new_bb, gt_bb))
+                pr_values.append(get_pr(new_bb, gt_bb, False))
+                n_pr_values.append(get_pr(new_bb, gt_bb, True))
```

Comment:
I think it is better to use keyword arg for `is_norm`:
```suggestion
                pr_values.append(get_pr(new_bb, gt_bb, is_norm=False))
                n_pr_values.append(get_pr(new_bb, gt_bb, is_norm=True))
```